src/views/workflow-history/workflow-history-event-status-badge/workflow-history-event-status-badge.tsx

```diff
@@ -9,19 +10,37 @@ import {
   MdReportGmailerrorred,
 } from 'react-icons/md';
 
+import getBadgeContainerSize from './helpers/get-badge-container-size';
 import getBadgeIconSize from './helpers/get-badge-icon-size';
 import {
   WORKFLOW_EVENT_STATUS,
   WORKFLOW_EVENT_STATUS_BADGE_SIZES,
 } from './workflow-history-event-status-badge.constants';
-import { styled } from './workflow-history-event-status-badge.styles';
+import {
+  styled,
+  overrides,
+} from './workflow-history-event-status-badge.styles';
 import type { Props } from './workflow-history-event-status-badge.types';
 
 export default function WorkflowHistoryEventStatusBadge({
   status,
+  statusReady,
```

Comment:
```suggestion
  isIncomplete,
```
I think this would be a better variable name since we are rendering a special icon if the event has incomplete information.